index.js

```diff
@@ -256,6 +256,10 @@
     };
   };
 
+  var compose3 = function(f, g, x) {
+    return f(g(x));
+  };
+
```

Comment:
Thanks for the explanation. I agree with your conclusion.

It's unfortunate that we have both `compose2` and `compose3`, but this doesn't bother me greatly.